src/main/resources/io/seqera/wave/spack/spack-builder-singularityfile.txt

```diff
@@ -20,27 +20,26 @@ EOF
     # import AWS environment
     . /aws-env.sh
 
-    # Copy and modify spack.yaml
-    sed -i -e 's;compilers:;compilers::;' \
-           -e 's;^ *flags: *{};    flags:\n      cflags: -O3\n      cxxflags: -O3\n      fflags: -O3;' \
-           /root/.spack/linux/compilers.yaml
-
     # Set up Spack environment
     export PATH=/opt/spack/bin:$PATH
     cd /opt/spack-env
```

Comment:
How was this `cd` working at all if the directory doesn't exist in the image?

Reply:
same reason as above, The directory gets created at line 3 of the Dockerfile by this `COPY` command:
```
COPY spack.yaml /opt/spack-env/spack.yaml
```